src/MusicPlaylistSecondary.java

```diff
@@ -0,0 +1,169 @@
+
+import components.map.Map;
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/**
+ * Layered implementations of secondary methods for {@code SimpleWriter}.
+ *
+ * @author Nijaa Nishanth
+ *
+ */
+public abstract class MusicPlaylistSecondary implements MusicPlaylist {
+
+    /*
+     * Playlist
+     */
+    private Queue<Map<String, String>> playlist;
```

Comment:
Abstract classes should not have fields. If you need this object, make sure a kernel returns it (or the information you need).